server/middleware/maintenance.js

```diff
@@ -1,11 +1,37 @@
-const logger = require('../../log')
+const moment = require('moment')
+const config = require('../../config')
+const { isNullOrEmpty } = require('../helpers/utilities')
 
-module.exports = (req, res) => {
-  logger.info('GET maintenance view')
-  res.render('pages/maintenance', {
-    employeeName: req.user.employeeName,
-    csrfToken: res.locals.csrfToken,
-    hmppsAuthMFAUser: req.hmppsAuthMFAUser,
-    authUrl: req.authUrl,
-  })
+module.exports = (
+  { user: { employeeName = '' } = { employeeName: '' }, hmppsAuthMFAUser = false, authUrl = '' },
+  res,
+  next,
+) => {
+  try {
+    const {
+      maintenance: { start, end },
+    } = config
+    if (!isNullOrEmpty(start) && !isNullOrEmpty(end)) {
```

Comment:
Is this checking it `start` and `end` are not null values, will this have the same functionality if you use the boolean function instead of a double negative

`if (Boolean(start) && Boolean(end))`

Reply:
Interesting question. I actually think a `if(start && end)` would suffice.
If you did want to be belt and braces about it, we should check if this was a valid date String.
Everything in between is kinda pointless. The `isNullOrEmpty` function strips out white space characters and other things, but ultimately everything other than a valid date string will cause moment to return `invalid date`.

Reply:
This was lift and shift btw. There is an opportunity here to simplify the code base though and remove what seems to be an unnecessary helper.